docs/concepts/metrics/available_metrics/context_precision.md

```diff
@@ -87,9 +79,38 @@ Output
 0.49999999995
 ```
 
+## LLM Based Context Precision
+
+The following metrics uses LLM to identify if a retrieved context is relevant or not.
+
+### Context Precision without reference
+
+The `LLMContextPrecisionWithoutReference` metric can be used without the availability of a reference answer. To estimate if the retrieved contexts are relevant, this method uses the LLM to compare each chunk in `retrieved_contexts` with the `response`.
+
+#### Example
```

Comment:
There's an example above which is identical. Don't suppose we need another one which shows exact same thing.

Reply:
My response to your other comment answers this question.